services/base-service/spa/config.js

```diff
@@ -32,10 +32,14 @@ config.client = {
         appName : 'aggie-experts',
         appOwner : 'digital'
       }
-    }
+    },
+    experts: {
+      is_public : (process.env.EXPERTS_IS_PUBLIC === "false") || true
```

Comment:
@UcDust this is just an example, The login redirect needs changing, and I don't know what else